.env.example

```
@@ -57,6 +59,18 @@ AXON_OPENAI_MODEL=
 AXON_OPENAI_API_KEY=
 
+# Codex app-server backend settings. This is not an OpenAI-compatible HTTP
+# endpoint; Axon spawns `codex app-server` per completion over stdio.
+# AXON_CODEX_HOME is optional when the default Codex home has auth. The child
+# uses an isolated throwaway CODEX_HOME and does not load user hooks, MCP
+# servers, apps, or skills. Bare OPENAI_API_KEY is ignored by Axon config but
+# may be forwarded only into this isolated Codex child as an optional auth path.
+AXON_CODEX_CMD=
+AXON_CODEX_HOME=
+AXON_SYNTHESIS_CODEX_MODEL=
+AXON_CODEX_MODEL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AXON_CODEX_MODEL key should go before the AXON_SYNTHESIS_CODEX_MODEL key

(UnorderedKey)